### ISSUE TITLE
pom.xml should be updated to specify the latest version of SqlRender

### ISSUE THREAD

**Issue by leeevans** [OHDSI/WebAPI#226]
The latest version of SqlRender is:
1.3.0-SNAPSHOT
See:
https://github.com/OHDSI/SqlRender/blob/master/pom.xml

The WebAPI pom.xml is still referencing the older version of SqlRender:

`<dependency>
      <groupId>org.ohdsi.sql</groupId>
      <artifactId>SqlRender</artifactId>
      <version>1.0.0-SNAPSHOT</version>
</dependency>
`

The fix is to update the WebAPI pom.xml SqlRender dependency.

**Comment by chrisknoll** [comment]
Hello @leeevans , 
  This change is part of PR #210, but is under review right now.  If this is something that can't wait, maybe we can pull together a separate PR just for this change.  Let us know your preference.

**Comment by leeevans** [comment]
Hi @chrisknoll a separate PR sounds good. 

The latest SqlRender includes the Impala support that Tom White created for the OHDSI Hadoop work group. 

Within that work group I've volunteered to test Atlas on the new OHDSI Cloud Impala instance and Malcolm McRoberts has volunteered to test it at his local site.

**Comment by chrisknoll** [comment]
Also you could try running these commands on a local environment to test:

```
git checkout -b myl-google-master master
git pull https://github.com/myl-google/WebAPI.git master
```

(These are the commands given by the github PR to see the changes). If you do this you could test the changes without a separate PR.  And, as a benefit, any issues you find could be ammendments to the PR for #210 without having two competing PRs for the same feature. I believe #210 is exactly the  functionality you're trying to test, so maybe we should keep this as just a single PR?

**Comment by leeevans** [comment]
Thanks for the quick response @chrisknoll 

#210 is for implementing Google Bigquery support which Taha is leading.

The functionality I'm looking to test is from PR #213 (Cloudera Impala support) which was already merged into webapi and depends on the latest version of sqlrender (Impala support is PR OHDSI/SqlRender#81 which was already merged into sqlrender).

Another way to look at it is that the latest release of WebAPI is currently running with an out of date version of SqlRender, so the request here is to update the old dependency in the pom.xml so it will pull the correct latest version.

**Comment by chrisknoll** [comment]
Ok, @leeevans , I'm sorry, I got my wires crossed on that.  We can set up a PR for just the SqlRender update and then the #210.

I'll let you know when this is available.

**Comment by t-abdul-basser** [comment]
@chrisknoll Yes, as @leeevans  said, I am working with @myl-google to integrate changes associated with PR #210 adding (partial) BigQuery support. I will address any conflicts associated with that PR. 

I do second what @leeevans originally suggested in this issue, however, i.e. that the version of SqlRender be updated in the POM.

**Comment by leeevans** [comment]
@chrisknoll thanks for making the change in Master. 

Can you also make the Sqlrender version pom.xml change in the 'Released' branch? 
The Jenkins job for OHDSI cloud deployment builds from the the Released branch. 

Sorry, I should have specified the branch in my original request.

**Comment by chrisknoll** [comment]
The released branch would require a new release to be created.  We don't make commits to the released branch.  I think I noticed that commits were made directly ontop of the release branch but this interferes with our release process.  The release process is as follows:

1. Get master into the final release state
2. In github, create a new release, based on master, and enter a new tag value (such as v2.1.1) This includes creating release notes.
3. Publish the release, which creates the tag.
4. Move the released branch using the following git command based on the SO article https://stackoverflow.com/questions/5471174/move-branch-pointer-to-different-commit-without-checkout: 
```
git update-ref refs/heads/released {tag} (ie: v2.1.1)
git push -f released
```

The step 4 moves the released branch directly without needing a commit.

I'll create a hotfix that just makes this change under a 2.1.1 release.

**Comment by chrisknoll** [comment]
2.1.1 was released with the POM update.
